RELEASES.md

```diff
@@ -2414,6 +2414,10 @@ Language
 - [Visibility modifiers (e.g. `pub`) are now syntactically allowed on trait items and
   enum variants.][66183] These are still rejected semantically, but
   can be seen and parsed by procedural macros and conditional compilation.
+- [You can now define a rust extern "C" function with `Box<T>` and use `T*` as corresponding
+  type on the C side.][62514] Please see [the documentation][box-memory-layout] for more information.
```

Comment:
```suggestion
  type on the C side.][62514] Please see [the documentation][box-memory-layout] for more information,
  including the important caveat about preferring to avoid `Box<T>` in Rust signatures for functions defined in C.
```

From the documentation:

> Important. At least at present, you should avoid using `Box<T>` types for functions that are defined in C but invoked from Rust. In those cases, you should directly mirror the C types as closely as possible. Using types like `Box<T>` where the C definition is just using `T*` can lead to undefined behavior, as described in rust-lang/unsafe-code-guidelines#198.

Seems good to call out directly, I think.

Reply:
That's why I thought it necessary to at least include the link to the docs. Was not sure if including the warning would make the bullet point too big